docs/dev/client.rst

```diff
@@ -0,0 +1,79 @@
+############
```

Comment:
This is a user guide for the Nublado client right? In that case, it doesn't belong in the Nublado developer documentation, but instead as a separate part of the Nublado docs.

Reply:
OK, I'll hoist it up a level.